terraform.tfvars

```diff
@@ -0,0 +1,4 @@
+env_code= "f2"
+vpc_cidr = "10.0.0.0/16"
+public_cidr=["10.0.0.0/24","10.0.1.0/24","10.0.2.0/24"]
+private_cidr=["10.0.100.0/24","10.0.101.0/24","10.0.103.0/24"]
```

Comment:
could you please, add a newline at the end of file

Reply:
Added a new line at the end of the file